taxcalc/functions.py

```diff
@@ -540,9 +540,17 @@ def TaxGains(e00650, c01000, c04800, c23650, p23250, e01100, e58990,
         else:
             addtax = 0.
 
+        if c24540 > CG_thd_xtr[MARS - 1]:
+            addtax = (CG_rt_xtr - CG_rt3) * c24517 + addtax
+        elif c24540 <= CG_thd_xtr[MARS - 1] and _taxinc > CG_thd_xtr[MARS - 1]:
+            addtax = addtax + (CG_rt_xtr - CG_rt3) * \
+             min(_dwks21, _taxinc - CG_thd_xtr[MARS - 1])
```

Comment:
I think this is the offending line. One way to make this PEP8 compliant would be to format this line in the same way that lines 538-539 are formatted.